.circleci/config.yml

```diff
@@ -2000,7 +2003,8 @@ workflows:
       - static_analysis:
           requires: [ 'Prepare Code' ]
           name: "Static Analysis 80"
-          docker_image: circleci/php:8.0
+          docker_image: cimg/php:8.0
```

Comment:
We have to parametrize it because the change only applies to the composer version on PHP 8 images, not on 7.1.